deploy/crds/security.everoute.io_endpoints.yaml

```diff
@@ -33,34 +49,44 @@ spec:
           metadata:
             type: object
           spec:
+            description: Spec contains description of the endpoint
             properties:
               reference:
+                description: Reference of an endpoint, also the external_id of an
+                  ovs interface. We map between endpoint and ovs interface use the
+                  Reference.
                 properties:
                   externalIDName:
+                    description: ExternalIDName of an endpoint.
                     type: string
                   externalIDValue:
+                    description: ExternalIDValue of an endpint.
                     type: string
                 required:
                 - externalIDName
                 - externalIDValue
                 type: object
               vid:
+                description: VID describe the endpoint in which vlan or vxlan
```

Comment:
Use the vid represent the VLAN and VxLAN seems not proper, the VLAN range is 0～ 2^12, but VxLAN ID is larger 0～2^24. In current phase we don't support vxlan, so we can remove the vxlan in the description.

Reply:
done